config/initializers/cors.rb

```diff
@@ -0,0 +1,17 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'localhost:3005'
```

Comment:
- [ ] This CORS origin policy will block the majority of your users, unless they use port 3005 to run the frontend.

I tried to create a user with the default 3001 port and this happened:

![image](https://github.com/billymohajeri/Final-Group-Capstone-Backend/assets/57421823/6522131e-8112-4220-901b-9787a69e1a99)

Kindly, either change this line 10 to "*" so any port is good, or include on your frontend readme instructions about how to run the frontend on port 3005. 👍